egs/snips/v1/local/gen_topo.pl

```diff
@@ -0,0 +1 @@
+../../../mobvoi/v1/local/gen_topo.pl
```

Comment:
I think rather than in  'local', these scripts that you want to share should go in a directory `wakeword`, e.g. in /scripts/wakeword, and you can put a soft link to them. 
In future I'd like shared scripts to go there.